### PR TITLE
add ability to negate filters

### PR DESCRIPTION
**Pull request by rbren**
Note: minor behavior change. Instead of making people fill out _all_ filters, we only make them fill out at least one.

### CODE REVIEW

File: prune-releases.sh [hunk 1]
```diff
@@ -56,7 +64,7 @@ while [ "$1" != "" ]; do
     shift
 done
 
-if [ -z "$older_than_filter" -o -z "$release_filter" -o -z "$namespace_filter" ]; then
+if [[ -z "$older_than_filter" && -z "$release_filter" && -z "$namespace_filter" && -z "$namespace_negate_filter" && -z "$release_negate_filter" && -z "$max_releases_to_keep" ]]; then
```
Comment (mhoss019) [inline prune-releases.sh:67]: What does -z do?

Comment (mhoss019) [reply prune-releases.sh:67]: nvm. Had trouble googling the right words.

**Review by mhoss019 — APPROVED** [review]
I think I understand and it looks okay to me.

File: prune-releases.sh (hunk 1 above)
Comment (rbren) [reply prune-releases.sh:67]: Hah the minus filter always gets me - I think you can put it in quotes to not make it a minus sign

**Review by sudermanjr — APPROVED** [review]

**Pull request merged**